mysql-test/collections/disabled.def

```diff
@@ -103,6 +103,7 @@ rpl_nogtid.rpl_binlog_format_errors               : BUG#29776083 EXPLICIT BINLOG
 # rpl suite tests
 rpl.rpl_io_thd_wait_for_disk_space_stress : BUG#23581287 Disabled until bug is fixed.
 rpl.rpl_writeset_add_unique_key : Bug#33134835 RPL_WRITESET_ADD_UNIQUE_KEY FAILS SPORADICALLY
+rpl.rpl_gtid_while_super_read_only : Bug#0 Fails on PS, even on earlier versions
```

Comment:
Does it fail on upstream as well? If not, please create a Jira ticket for this (if we already don't have it)

Reply:
Jira ticket created.